Bug: build fails with "error: ‘temp’ may be used uninitialized in this function"

**Describe the bug**
assimp from master branch fails to build on openSUSE Leap Linux with the following error:

```
In file included from /home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest-matchers.h:48:0,
                 from /home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/internal/gtest-death-test-internal.h:46,
                 from /home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest-death-test.h:43,
                 from /home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest.h:60,
                 from /home/user/src/vsgFramework-build/components/assimp-src/test/unit/UnitTestPCH.h:50,
                 from /home/user/src/vsgFramework-build/components/assimp-src/test/unit/AbstractImportExportBase.h:45,
                 from /home/user/src/vsgFramework-build/components/assimp-src/test/unit/ImportExport/MDL/utMDLImporter_HL1_ImportSettings.cpp:46:
/home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest-printers.h: In static member function ‘static void utMDLImporter_HL1_ImportSettings::expect_global_info_eq(const aiScene*, std::initializer_list<std::pair<T, const char*> >) [with T = int]’:
/home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest-printers.h:216:5: error: ‘temp’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
     *os << value;
     ^
/home/user/src/vsgFramework-build/components/assimp-src/test/unit/ImportExport/MDL/utMDLImporter_HL1_ImportSettings.cpp:219:15: note: ‘temp’ was declared here
             T temp;
               ^~~~
In file included from /home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest-matchers.h:48:0,
                 from /home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/internal/gtest-death-test-internal.h:46,
                 from /home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest-death-test.h:43,
                 from /home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest.h:60,
                 from /home/user/src/vsgFramework-build/components/assimp-src/test/unit/UnitTestPCH.h:50,
                 from /home/user/src/vsgFramework-build/components/assimp-src/test/unit/AbstractImportExportBase.h:45,
                 from /home/user/src/vsgFramework-build/components/assimp-src/test/unit/ImportExport/MDL/utMDLImporter_HL1_ImportSettings.cpp:46:
/home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest-printers.h: In static member function ‘static void utMDLImporter_HL1_ImportSettings::expect_global_info_eq(const aiScene*, T, const char*) [with T = int]’:
/home/user/src/vsgFramework-build/components/assimp-src/contrib/gtest/include/gtest/gtest-printers.h:216:5: error: ‘temp’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
     *os << value;
     ^
/home/user/src/vsgFramework-build/components/assimp-src/test/unit/ImportExport/MDL/utMDLImporter_HL1_ImportSettings.cpp:208:11: note: ‘temp’ was declared here
         T temp;
           ^~~~
```
**To Reproduce**
Steps to reproduce the behavior:
1. clone git repo https://github.com/vsg-dev/vsgFramework
2. enter the directory where the repo has been cloned
3. run cmake -B ../vsgFramework-build && cmake --build ../vsgFramework-build
4. See error

**Expected behavior**
The build should not fail

**Platform (please complete the following information):**
 - OS: openSUSE Leap
 - Version 15.3

--- a/test/unit/ImportExport/MDL/utMDLImporter_HL1_ImportSettings.cpp
+++ b/test/unit/ImportExport/MDL/utMDLImporter_HL1_ImportSettings.cpp
@@ -205,7 +205,7 @@ private:
             const char *key_name) {
         aiNode *global_info = get_global_info(scene);
         EXPECT_NE(nullptr, global_info);
-        T temp;
+        T temp = 0;
         EXPECT_TRUE(global_info->mMetaData->Get(key_name, temp));
         EXPECT_EQ(expected_value, temp);
     }
@@ -216,7 +216,7 @@ private:
         aiNode *global_info = get_global_info(scene);
         EXPECT_NE(nullptr, global_info);
         for (auto it = p_kv.begin(); it != p_kv.end(); ++it) {
-            T temp;
+            T temp = 0;
             EXPECT_TRUE(global_info->mMetaData->Get(it->second, temp));
             EXPECT_EQ(it->first, temp);
         }

